hack/tests/scaffolding/scaffold-memcached.go

```diff
@@ -0,0 +1,285 @@
+		"--api-version=cache.example.com/v1alpha1",
+		"--kind=Memcached").CombinedOutput()
+	if err != nil {
+		log.Fatalf("Error: %v\nCommand Output: %s\n", err, string(cmdOut))
+	}
+
+	tmplFiles := map[string]string{
+		filepath.Join(localSDKPath, "example/memcached-operator/memcached_controller.go.tmpl"): "pkg/controller/memcached/memcached_controller.go",
+		filepath.Join(localSDKPath, "test/e2e/_incluster-test-code/main_test.go"):              "test/e2e/main_test.go",
+		filepath.Join(localSDKPath, "test/e2e/_incluster-test-code/memcached_test.go"):         "test/e2e/memcached_test.go",
+	}
+	for src, dst := range tmplFiles {
+		if err := os.MkdirAll(filepath.Dir(dst), fileutil.DefaultDirFileMode); err != nil {
+			log.Fatalf("Could not create template destination directory: %s", err)
+		}
+		cmdOut, err = exec.Command("cp", src, dst).CombinedOutput()
+		if err != nil {
+			log.Fatalf("Error: %v\nCommand Output: %s\n", err, string(cmdOut))
+		}
+	}
+
+	memcachedTypesFile, err := ioutil.ReadFile("pkg/apis/cache/v1alpha1/memcached_types.go")
+	if err != nil {
+		log.Fatal(err)
+	}
+	memcachedTypesFileLines := bytes.Split(memcachedTypesFile, []byte("\n"))
+	for lineNum, line := range memcachedTypesFileLines {
+		if strings.Contains(string(line), "type MemcachedSpec struct {") {
+			memcachedTypesFileLinesIntermediate := append(memcachedTypesFileLines[:lineNum+1], []byte("\tSize int32 `json:\"size\"`"))
+			memcachedTypesFileLines = append(memcachedTypesFileLinesIntermediate, memcachedTypesFileLines[lineNum+3:]...)
+			break
+		}
+	}
+	for lineNum, line := range memcachedTypesFileLines {
+		if strings.Contains(string(line), "type MemcachedStatus struct {") {
+			memcachedTypesFileLinesIntermediate := append(memcachedTypesFileLines[:lineNum+1], []byte("\tNodes []string `json:\"nodes\"`"))
+			memcachedTypesFileLines = append(memcachedTypesFileLinesIntermediate, memcachedTypesFileLines[lineNum+3:]...)
+			break
+		}
+	}
+	if err := os.Remove("pkg/apis/cache/v1alpha1/memcached_types.go"); err != nil {
+		log.Fatalf("Failed to remove old memcached_type.go file: (%v)", err)
+	}
+	err = ioutil.WriteFile("pkg/apis/cache/v1alpha1/memcached_types.go", bytes.Join(memcachedTypesFileLines, []byte("\n")), fileutil.DefaultFileMode)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	log.Print("Generating k8s")
+	cmdOut, err = exec.Command("operator-sdk", "generate", "k8s").CombinedOutput()
+	if err != nil {
+		log.Fatalf("Error: %v\nCommand Output: %s\n", err, string(cmdOut))
+	}
+
+	log.Print("Pulling new dependencies with go mod")
+	cmdOut, err = exec.Command("go", "build", "./...").CombinedOutput()
+	if err != nil {
+		log.Fatalf("Command \"go build ./...\" failed: %v\nCommand Output:\n%v", err, string(cmdOut))
+	}
+
+	operatorYAML, err := ioutil.ReadFile("deploy/operator.yaml")
+	if err != nil {
+		log.Fatalf("Could not read deploy/operator.yaml: %v", err)
+	}
+	if *imageName == "" {
+		*imageName = "quay.io/example/memcached-operator:v0.0.1"
+		if err != nil {
```

Comment:
What error are we checking here?

Reply:
I have no idea lol. It was in the original code that this was migrated from: https://github.com/operator-framework/operator-sdk/blob/master/test/e2e/memcached_test.go#L555L:L559

I'll remove it as it's unnecessary.